pkg/controller/node/controller_utils.go

```diff
@@ -36,17 +36,28 @@ import (
 )
 
 // This function is expected to get a slice of NodeReadyConditions for all Nodes in a given zone.
+// The zone is considered:
+// - fullyDisrupted if there're no Ready Nodes,
+// - partiallyDisrupted if more than 1/3 of Nodes (no less than 3) are not Ready,
```

Comment:
First done, second discussed f2f (it's a 3-value logic, so NotReady != not Ready)